src/platform_impl/linux/event_loop.rs

```diff
@@ -421,6 +422,33 @@ impl<T: 'static> EventLoop<T> {
                 | EventMask::SCROLL_MASK,
             );
 
+            const TITLEBAR_HEIGHT: f64 = 30.0;
```

Comment:
this feels like a magic number, shouldn't we query and check the actual height of the header bar?

Reply:
I’ve tried the following:
```rust
let full_height = window.allocated_height();
let inner_height = window.size().1;
let header_height = full_height - inner_height;
```
this results in significant gaps in the length of the existing header.

Reply:
Using `window.titlebar()`, I was able to retrieve the exact size of the header. 
I modified the code to get the title bar height without using magic numbers.